### PR TITLE
feat(tools/tokens): add preset for adeo

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a breaking change?

read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)

- [ ] Yes
- [x] No

## Describe the changes

Add a preset config to manage Adeo's colors and properties

Github issue number or Jira issue URL: **MZC-200**

### CODE REVIEW

File: src/docs/GetStarted/Developers/PresetAdeo/index.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+```js
+module.exports = {
+  preset: 'adeo'
+}
+```
+
+Once this option is added to the `mozaic.config.js` file, you must restart the token compilation; as described in the ["Compile Tokens" chapter](/GetStarted/Developers/CustomizeTokens/#3---compiling-tokens) of the customization documentation.
+
+> **CAUTION:** When recompiling tokens, several warning messages beginning with "Collision detected at:" are likely to appear. This is quite normal, it simply indicates that the default values of Mozaic have been overloaded by those of Adeo. So you have nothing more to do.
```
Comment (gael-boyenval) [inline src/docs/GetStarted/Developers/PresetAdeo/index.mdx]: overloaded ? overridden ?

File: src/docs/GetStarted/Developers/PresetAdeo/index.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Variable $preset
```
Comment (gael-boyenval) [inline src/docs/GetStarted/Developers/PresetAdeo/index.mdx]: don't need to document that as long as there is no visible / actionable effects.

File: src/docs/GetStarted/Developers/PresetAdeo/index.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+In order to activate the **Adeo preset**, you must create at the root of your project a file `mozaic.config.js` if it does not already exist, and add to this file the following code :
+
+```js
+module.exports = {
+  preset: 'adeo'
+}
+```
+
+Once this option is added to the `mozaic.config.js` file, you must restart the token compilation; as described in the ["Compile Tokens" chapter](/GetStarted/Developers/CustomizeTokens/#3---compiling-tokens) of the customization documentation.
```
Comment (gael-boyenval) [inline src/docs/GetStarted/Developers/PresetAdeo/index.mdx]: you need to define a path for built tokens also I presume. In that case this should be documented here.

File: src/docs/GetStarted/Developers/PresetAdeo/index.mdx (hunk 1 above)
Comment (tiloyi) [reply src/docs/GetStarted/Developers/PresetAdeo/index.mdx]: fixed

File: src/docs/GetStarted/Developers/PresetAdeo/index.mdx (hunk 2 above)
Comment (tiloyi) [reply src/docs/GetStarted/Developers/PresetAdeo/index.mdx]: fixed

File: src/docs/GetStarted/Developers/PresetAdeo/index.mdx (hunk 3 above)
Comment (tiloyi) [reply src/docs/GetStarted/Developers/PresetAdeo/index.mdx]: fixed

File: src/docs/GetStarted/Designers/PresetAdeo/index.mdx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: 'Preset Adeo'
+order: 6
+---
+
+> Mozaic is designed to be above all the design system of the LEROY MERLIN brand. However, we also offer you the possibility to extend Mozaic to meet specific customization needs.
+
+## Adeo color palette
+
+It is in this context that we have set up the possibility of customizing Mozaic to meet the needs of LEROY MERLIN's parent company: ADEO.
```
Comment (nherwegh) [inline src/docs/GetStarted/Designers/PresetAdeo/index.mdx]: We have set Mozaic to be customisable to meet the needs of LEROY MERLIN's main brand: ADEO.

File: src/docs/GetStarted/Designers/PresetAdeo/index.mdx [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+Indeed, Mozaic provides you with a designer kit, first the LEROY MERLIN kit, and in parallel the ADEO kit.
```
Comment (nherwegh) [inline src/docs/GetStarted/Designers/PresetAdeo/index.mdx]: this sentence is weird too

File: src/docs/GetStarted/Designers/PresetAdeo/index.mdx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+title: 'Preset Adeo'
+order: 6
+---
+
+> Mozaic is designed to be above all the design system of the LEROY MERLIN brand. However, we also offer you the possibility to extend Mozaic to meet specific customization needs.
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply src/docs/GetStarted/Designers/PresetAdeo/index.mdx]: @nherwegh I just wanted to point out that Mozaic is first and foremost a product for LM. Do you have a better way to say this ?

[the remark this answers is not in the document]
Comment (nherwegh) [reply src/docs/GetStarted/Designers/PresetAdeo/index.mdx]: @tiloyi, maybe this as an introduction : Mozaic is first thought to answer to LEROY MERLIN's brand guidelines but customisation is also allowed.

File: src/docs/GetStarted/Designers/PresetAdeo/index.mdx (hunk 5 above)
Comment (nherwegh) [reply src/docs/GetStarted/Designers/PresetAdeo/index.mdx]: With Mozaic, we provide you two different UI kits. One that is branded LEROY MERLIN and another one branded Adeo. Feel free to download the folder and explore these two UI kits.

File: src/docs/GetStarted/Designers/PresetAdeo/index.mdx [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+## Adeo color palette
+
+It is in this context that we have set up the possibility of customizing Mozaic to meet the needs of LEROY MERLIN's parent company: ADEO.
+
+Indeed, Mozaic provides you with a designer kit, first the LEROY MERLIN kit, and in parallel the ADEO kit.
+
+Download and explore the folder to discover the different kits.
```
Comment (nherwegh) [inline src/docs/GetStarted/Designers/PresetAdeo/index.mdx]: delete this sentence ;)

File: src/docs/GetStarted/Designers/PresetAdeo/index.mdx [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+title: 'Preset Adeo'
+order: 6
+---
+
+> Mozaic is first thought to answer to LEROY MERLIN's brand guidelines but customisation is also allowed.
+
+## Adeo color palette
+
+We have set Mozaic to be customisable to meet the needs of LEROY MERLIN's group: ADEO.
+Adeo provides collaborative tools for its employees, and the visual identity in these tools is based on Adeo's brand colors.
```
Comment (nherwegh) [inline src/docs/GetStarted/Designers/PresetAdeo/index.mdx]: maybe remove the `and` and do two different sentences

**Pull request merged**